### ISSUE TITLE
Symfony.Formatting.ReturnOrThrow.Invalid false positive on anonymous function

### ISSUE THREAD

**Issue by Soullivaneuh** [djoos/Symfony-coding-standard#123]
Code sample:

```php
switch ($incident->getState()) {
    case Incident::STATE_OPENED:
        $this->considerIncident($incident);
        break;
    case Incident::STATE_CONSIDERED:
        $this->resolveIncident($incident);
        break;
    case Incident::STATE_RESOLVED:
        $publicComments = $incident->getComments()->filter(static function (IncidentComment $comment): bool {
            return true === $comment->getCustomerVisible();
        });
        $this->closeIncident($incident);
        break;
}
```

I have:

```
Do not use else, elseif, break after if and case conditions which return or throw something (Symfony.Formatting.ReturnOrThrow.Invalid)
```

Pointing on `return true === $comment->getCustomerVisible();` of the anonymous function.

It does not seems to make any sense here.

**Comment by stof** [comment]
This also happens for me when having a closure inside a `if` statement.

**Comment by stof** [comment]
This is the same than https://github.com/djoos/Symfony-coding-standard/issues/118

**Comment by istrof** [comment]
https://github.com/djoos/Symfony-coding-standard/pull/126 fixes that.

**Comment by djoos** [comment]
#126 merged, thanks @istrof!
